content/configuration/dev-server.md

```diff
@@ -74,6 +74,24 @@ allowedHosts: [
 ]
 ```
 
+To use this option with the CLI pass the `--allowed-hosts` option a comma-delimited string.
+
+```
+webpack-dev-server --entry /entry/file --output-path /output/path --allowed-hosts .host.com,host2.com
+```
+
+## `devServer.bonjour`
+
+This option broadcasts the server via ZeroConf networking on start
+
+```js
+bonjour: true
+```
+
+Usage via the CLI
+```
```

Comment:
You need to add a line break between each fenced code block and the paragraph above it.